lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -214,6 +214,77 @@ speed:
   config_set_append: "speed %s"
   default_value: "auto"
 
+stp_bpdufilter:
+  kind: string
```

Comment:
No. They all have some string enums.
please see the code on the puppet-module (which is in my fork only) for the available values.
For ex:
https://github.com/saichint/cisco-network-puppet-module/blob/feature/stp_global/lib/puppet/type/cisco_interface.rb#L521

Also I used Chris' example for channel_group for "default_value: false" (even though kind is 'int') because the entire CLIs for the attributes you mentioned here do not exist sometimes when they are in default.
https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/interface_channel_group.yaml#L16